armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -222,6 +228,23 @@ def interactCoupled(self, iteration):
 
         GlobalFluxInterface.interactCoupled(self, iteration)
 
+    def getTightCouplingValue(self):
+        if self.tightCouplingConvergeOn == "keff":
+            return self.r.core.p.keff
+        if self.tightCouplingConvergeOn == "power":
+            scaledCorePowerDistribution = []
+            for a in self.r.core.getChildren():
+                scaledPower = []
+                assemPower = sum(b.p.power for b in a)
+                for b in a:
+                    scaledPower.append(b.p.power / assemPower)
+
+                scaledCorePowerDistribution.append(scaledPower)
+
+            return numpy.array(scaledCorePowerDistribution, dtype=object)
```

Comment:
This branch is interesting. Since `keff` is hard-coded above, why is the `power` branch going to be hit, if ever?

Reply:
As of now, `power` won't get hit. The idea was to expose this as a user setting and use `keff` and `power` as the available options to converge on. Allow for some user flexibility. I just haven't gotten around to exposing it as a setting quite yet.